build/scripts/locate-vs.ps1

```diff
@@ -13,7 +13,7 @@ function Create-Directory([string[]] $path) {
 
 function Locate-LocateVsApi {
   $packagesPath = Locate-PackagesPath
-  $locateVsApi = Join-Path -path $packagesPath -ChildPath "RoslynTools.Microsoft.LocateVS\$locateVsApiVersion\lib\net46\LocateVS.dll"
```

Comment:
Correct on the latter. There is both the library version (for referencing from another app) and the Tool version (for referencing from Powershell, etc)